sdk/javascript/processor/index.js

```diff
@@ -155,7 +155,7 @@ class TransactionProcessor {
         .then(content => TpRegisterResponse.decode(content))
         .then(ack => {
           let {transactionFamilyName: familyName, version} = handler
-          let status = ack.status === 0 ? 'succeeded' : 'failed'
+          let status = ack.status === 1 ? 'succeeded' : 'failed'
```

Comment:
This should use the enum here, which would avoid the issue of having to correct this kind of magic number.